sandboxed_containers/sandboxed-containers-4.11-release-notes.adoc

```diff
@@ -0,0 +1,43 @@
+:_content-type: ASSEMBLY
+[id="sandboxed-containers-1-3-release-notes"]
+= {sandboxed-containers-first} {sandboxed-containers-version} release notes
+include::_attributes/common-attributes.adoc[]
+:context: sandboxed-containers-release-notes
+
+toc::[]
+
+[id="sandboxed-containers-1-3-about-this-release"]
+== About this release
+
+These release notes track the development of {sandboxed-containers-first} {sandboxed-containers-version} alongside Red Hat {product-title} {product-version}.
+
+This product is fully supported and enabled by default as of {product-title} 4.10.
+
+[id="sandboxed-containers-1-3-new-features-and-enhancements"]
+== New features and enhancements
+
+
+
+[id="sandboxed-containers-1-3-bug-fixes"]
+== Bug fixes
+
+
+
+[id="sandboxed-containers-1-3-known-issues"]
+== Known issues
+
+
+
+[id="sandboxed-containers-1-3-asynchronous-errata-updates"]
+== Asynchronous errata updates
+
+Security, bug fix, and enhancement updates for {sandboxed-containers-first} {product-version} are released as asynchronous errata through the Red Hat Network. All {product-title} {product-version} errata is https://access.redhat.com/downloads/content/290/[available on the Red Hat Customer Portal]. See the https://access.redhat.com/support/policy/updates/openshift[{product-title} Life Cycle] for more information about asynchronous errata.
+
+Red Hat Customer Portal users can enable errata notifications in the account settings for Red Hat Subscription Management (RHSM). When errata notifications are enabled, users are notified via email whenever new errata relevant to their registered systems are released.
```

Comment:
```suggestion
You can enable errata notifications in the account settings for Red Hat Subscription Management (RHSM) on the Red Hat Customer Portal. When enabled, you are notified by email whenever new errata relevant to your registered systems are released.
```

Using a direct voice, "you/your" instead of "users" makes the text a bit easier to read.

Reply:
Again, copied directly from OCP RN. Gonna leave this one for now.